rmf_fleet_adapter/test/phases/RequestLiftTest.cpp

```diff
@@ -89,6 +114,7 @@ SCENARIO_METHOD(MockAdapterFixture, "request lift phase", "[phases]")
       }
     }
 
+    /*
```

Comment:
We should check whether or not this test block has valid expectations. If the expectations are no longer valid, then we should delete it.

Reply:
I don't think this test block has valid expectations, as after the phase is canceled, an `END_SESSION` request should be sent according to the newly written test block. But this test block is still passing. Not sure if something is wrong or if the test is just checking the wrong thing.